src/main/resources/mediator.properties

```diff
@@ -1,11 +1,23 @@
 # Mediator Properties
 mediator.name=HFR-Mediator
 mediator.host=localhost
-mediator.port=3003
+mediator.port=3001
```

Comment:
Please update this to 3003, Port 3001 is already taken by another mediator

Reply:
Changes are up